src/SpecialOffer.java

```diff
@@ -0,0 +1,13 @@
+public class SpecialOffer {
+    Product product;
+    String description;
+    int time;
+    double discount;
+    SpecialOffer(Product p, String d, int t, double dis){
+        product = p;
+        description = d;
+        //Nie widziałem jak zapisać czas od do, więc po prostu wstawiłem tutaj liczbę dni.
```

Comment:
na teraz jest to ok. Jesli jestes ciekawy to poczytaj o klasie `LocalDate` : https://javastart.pl/kurs/java/java-podstawy-temat-42-daty-i-czas/lekcja/java-podstawy-daty-w-javie-8